content/storage-history/history.textile

```diff
@@ -0,0 +1,596 @@
+---
+title: History
+meta_description: "Learn about accessing message history with the history and rewind features"
+languages:
+ - javascript 
+ - java 
+ - python 
+ - php 
+ - ruby 
+ - nodejs 
+ - objc
+ - swift
+ - go 
+ - csharp
+---
+
+History refers to accessing previously sent messages, enabling you to retrieve both channel history and presence history. If "storage":/storage-history/storage is enabled messages will typically be stored for 24 for free accounts or 72 hours for paid accounts, otherwise Ably retains the last two minutes of message history.
+
+h2(#history-rewind). History vs rewind
+
+There are two ways of accessing your channel history: history or rewind. Both history and rewind need the channel to have "persistence enabled":/storage-history/storage. 
+
+History is ideal for when you want to retrieve any message within your "storage time frame":/storage-history/storage other than the last published message, enabling a longer, more complete context. Rewind is useful for getting an instant update of a channel, enabling an update from either a given number of messages or a previous point in time as a time interval.
```

Comment:
Sorry, I think this section is still a little confused. I also think these examples don't help because they're so similar. 

History: paginated results, maximum limit of 1000 messages, presence or regular messages, REST or realtime interface, custom defined start and end time (the end doesn't have to be the present), can call repeatedly. Also has untilAttach which should be mentioned.

Rewind: returns messages at point of attachment, limit of 100 messages, only regular messages, realtime interface only, X number of messages or X seconds/minutes earlier (essentially the "start" with the "end" always being the present)